news/2019-04-24-TWC-2019-Group-Stage-Recap.md

```diff
@@ -0,0 +1,82 @@
+
+### Mappool Statistics
+
+**This week's most picked map was [Morimori Atsushi + Verdammt - Le Fruit Defendu](https://osu.ppy.sh/beatmapsets/808011#taiko/1695684), having been played in 31 out of 48 matches!** [MiddleIsland - Delrio](https://osu.ppy.sh/beatmapsets/917573#taiko/1916185), [Warak - Cosmic Invaders](https://osu.ppy.sh/beatmapsets/765624#taiko/1978335) and [dj MAX STEROID - Arabian Rave Night](https://osu.ppy.sh/beatmapsets/592980#taiko/1254592) were all tied at 36 interactions (picked + banned) each, making them the most popular/influential maps this stage.
+
+**Conversely, [Shano x 40mP - Sanpo Michi](https://osu.ppy.sh/beatmapsets/62991#taiko/186452) holds the title of most banned map of the week, with 18 bans to 17 picks.** Understandably, it was clear that many teams didn’t want to risk tackling this devilishly tricky 120 BPM map on Hidden. Finally, [CyoucyoP feat.Hatsune Miku - End of the World](https://osu.ppy.sh/beatmapsets/215476#taiko/506018) received the least attention at 16 interactions (10 picks and 6 bans). It's a bit disappointing, but the map has made more than its fair share of appearances in past TWCs; it's certainly not the end of the world as the song would otherwise suggest.
+
+## <a name="matches" id="matches"></a>Tournament Matches
+
+Over the course of last weekend, we saw our fair share of surprise performances from newcomer and veteran teams alike. Since it would be impossible to go through every single match in detail, here are some of the highlights below!
+
+### Saturday
+
+**The first match on the stream saw the unseeded China squaring off against the top-seeded Australia, resulting in a nail-bitingly close result of 3 — 5!**. osu!mania extraordinaire and Team Captain [Wilben_chan](https://osu.ppy.sh/users/6659363) surprised us all with his skill on the drums, landing many high accuracy scores alongside [7jasmine](https://osu.ppy.sh/users/4112195) to secure an early lead of 2 — 0. Australia was taken by surprise, but remained focused and proceeded to turn the tide as the match went on. A monumental showing from Team China, and they would later prove their strength again by defeating the low seeded Sweden 5 — 3.
+
+**Italy vs Brazil was one of the most anticipated matches this weekend, ending in a score of 2 — 5 in favour of Brazil.** Both teams' main roster remained largely unchanged from the previous year, though both sides had improved considerably since. With the score hovering at 2 — 2, the match initially seemed dead even at first. However, it was Brazil's diversity in numbers that gave them the winning edge, which Italy couldn't respond to towards the latter half of the match. Nevertheless, both teams have made it through to the next stage, where they may possibly face each other again?
```

Comment:
```suggestion
**Italy vs Brazil was one of the most anticipated matches this weekend, ending in a score of 2-5 in favour of Brazil.** Both teams' main roster remained largely unchanged from the previous year, though both sides had improved considerably since. With the score hovering at 2-2, the match initially seemed dead even at first. However, it was Brazil's diversity in numbers that gave them the winning edge, which Italy couldn't respond to towards the latter half of the match. Nevertheless, both teams have made it through to the next stage, where they may possibly face each other again?
```